packages/botonic-plugin-contentful/bin/migrations/content-types/url-shorttext.ts

```diff
@@ -0,0 +1,28 @@
+import Migration, { MigrationContext } from 'contentful-migration'
+
+import {
+  createFieldKeywords,
+  createFieldName,
+  createFieldShortText,
+  createFieldButtons,
+  createFieldFollowUp,
+} from '../factories'
+
+function crateUrl(migration: Migration) {
+  const url = migration.createContentType('url')
+
+  // TODO: make them required when Massimo contents migrated
+  createFieldName(url, true)
+  createFieldKeywords(url, true)
+  createFieldShortText(url, true)
+}
```

Comment:
There's a typo in the function's name. It should be `createUrl`, right?
Also when is this function used?

Reply:
I added contentful cli callback export

Reply:
After the changes done I don't see anymore this file in this PR, is this correct?

Reply:
@asastre I renamed it, since the original filenames were chosen due to historical reasons. It's now at url-image.ts